packages/components/TranscriptCard/index.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+import Alert from 'react-bootstrap/Alert';
+import Spinner from 'react-bootstrap/Spinner';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faTrash,
+  faCheck,
+  faExclamationTriangle,
+  faPen
+} from '@fortawesome/free-solid-svg-icons';
+
+const TranscriptCard = ({ transcriptItem, handleEdit, handleDelete }) => {
+
+  const handleDeleteClick = () => {
+    const confirmDeleteText = "Click OK if you wish to delete or cancel if you don't";
+    const cancelDeleteText = 'All is good, it was not deleted';
+
+    const confirmationPrompt = confirm(confirmDeleteText);
+
+    if (confirmationPrompt) {
+      return handleDelete ? handleDelete(transcriptItem.id) : alert(cancelDeleteText);
+    }
+  };
+
+  const handleEditClick = () => {
+    handleEdit(transcriptItem.id);
+  };
+
+  const setStatus = () => {
+    if (transcriptItem.status === 'error') {
+      return 'danger';
+    };
+    if (transcriptItem.status === 'in-progress') {
+      return 'info';
+    };
+    if (transcriptItem.status === 'done') {
+      return 'success';
+    };
+  };
+
```

Comment:
Do you need to set status to something else? Can we just use the status as is for subsequent functions?

Reply:
i.e. You could just use 'done' to set Badge `Success`

Reply:
Cool beans - good suggestion.